File: Week_03/high-card-dealer-js/src/Player.js
import Hand from './Hand.js'

class Player {
  constructor(name, hand = new Hand()) {
    this.name = name;
    this.hand = hand;
  };

  getName() {
    return this.name;
  };

  setName(name) {
    this.name = name;
  };
}

export default Player;

File: Week_14/java-team-roster/src/main/java/Player.java
public class Player {
  private String name;
  private String position;
  private String team;

  public Player(String position, String name) {
    this.name = name;
    this.position = position;
    this.team = "";
  }

  public String getName() { return name; }
  public void setName(String name) { this.name = name; }
  public String getPosition() { return position; }
  public void setPosition(String position) { this.position = position; }
  public String getTeam() { return team; }
  public void setTeam(String team) { this.team = team; }
}
